testTypeIperf.py

```diff
@@ -125,6 +125,11 @@ def cmd_line_args(self, *, for_template: bool = False) -> list[str]:
     def _create_setup_operation_get_cancel_action_cmd(self) -> str:
         return f"killall {IPERF_EXE}"
 
+    def _get_server_listen_protocol(self) -> str:
```

Comment:
this makes mypy sufficiently happy (for good reasons).

I still think that the overwritten function signature should be `_get_server_listen_protocol(self) -> Optional[str]:` to match the parent's signature. Even if our implementation in the subclass never actually returns `None`...

Reply:
done